addOns/selenium/src/main/javahelp/org/zaproxy/zap/extension/selenium/resources/help/contents/intro.html

```diff
@@ -10,8 +10,13 @@ <H1>Selenium</H1>
 	<p>The Selenium add-on provides WebDrivers, for other add-ons, to invoke and remotely
 		control web browsers. It's also bundled the HtmlUnit web browser, an headless Java based
 		web browser.
+		
+	<p>It adds a 'Selenium' script type - these scripts are invoked whenever selenium launches a browser,
+	for example for the Ajax Spider or for manual browsing. 
+	
 	<p>The add-on also adds a right click option to ZAP panels that list URLs (such at the History and Sites tree)
-	which allows you to open the selected URL in any of the supported browsers.
+	which allows you to open the selected URL in any of the supported browsers. These scripts have access to the 
+	launched browser and can run JavaScript commands on it.
```

Comment:
- `These scripts` → `The Selenium scripts`?
 - Shouldn't we say this when describing the script type above? Also, should we be more generic? e.g. `...and can interact with it, for example, run JavaScript scripts, access URLs, fill forms, click buttons...`.

Reply:
Done.